### PR TITLE
Add the new key format for Aggregation Service

### PR DESCRIPTION
**Pull request by linnan-github**
@johnivdel could you PTAL? Thanks!

### CODE REVIEW

File: AGGREGATE.md [hunk 1]
```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
+The  processing origin will respond with a set of keys and a cache control header which dictates how long to store the keys for. The browser will use these keys for the max-age as specified by the header. The scheme of the JSON encoded public keys is as follows:
```
Comment (alexmturner) [inline AGGREGATE.md:190]: optional: s/and a cache control header which dictates/which will be stored according to standard HTTP caching rules, i.e. using Cache-Control headers to dictate/
optional: s/max-age/freshness lifetime/

Do we want to mention that the browser might want to enforce minimum/maximum freshness lifetimes?

File: AGGREGATE.md [hunk 2]
```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
...
+
+
+```
+{
+  "version": "<format version>",
+  "keys": [
+    {
+      "id": "<arbitrary string identifying the key>",
```
Comment (alexmturner) [inline AGGREGATE.md:198]: can we specify a maximum string length here, e.g. up to 128 chars?

Comment (linnan-github) [reply AGGREGATE.md]: Fixed. Thanks!

File: AGGREGATE.md (hunk 1 above)
Comment (linnan-github) [reply AGGREGATE.md]: Updated based on suggestions. Thanks both.

File: AGGREGATE.md [hunk 3]
```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
+The processing origin will respond with a set of keys which will be stored according to standard HTTP caching rules, i.e. using Cache-Control headers to dictate how long to store the keys for (e.g. following the [freshness lifetime](https://datatracker.ietf.org/doc/html/rfc7234#section-4.2)). The browser could enforce maximum/minimum lifetimes of stored keys to encourage faster key rotation and/or mitigate bandwidth usage. The scheme of the JSON encoded public keys is as follows:
```
Comment (johnivdel) [inline AGGREGATE.md:190]: Could we add a bit more color about how these keys gets used to the start of this paragraph.

The browser will encrypt payloads just before the report is sent by requesting public keys for the processing origin. `keys.json` will be fetched with an un-credentialed request.

Let's also add a note below the schema like:

Note: The browser may need some mechanism to ensure that the same set of keys are delivered to different users.

**Comment by johnivdel** [comment]
lgtm % comment

File: AGGREGATE.md (hunk 3 above)
Comment (linnan-github) [reply AGGREGATE.md]: Thanks John. Updated per your suggestion.

**Review by johnivdel — APPROVED** [review]

**Comment by alexmturner** [comment]
lgtm too

**Pull request merged**